UberLoggerChannel.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections;
+using System.Collections.Generic;
+using UberLogger;
+using UnityEngine;
+
+/// <summary>
+/// Wraps access to a named channel in a class so that it can be used without having to
+/// type the channel name each time to enforcing channel name checking at compile-time.
+/// </summary>
+public class UberLoggerChannel
+{
+    private string _channelName;
```

Comment:
Slightly to my regret, the coding standard is ChannelName for private member variables that aren't shadowed by a property.